docs/setup/settings.asciidoc

```diff
@@ -366,10 +366,10 @@ used by the tile service. Specify the position of the subdomain the URL with
 the token `{s}`.
 
 | [[tilemap-url]] `map.tilemap.url:` {ess-icon}
- | The URL to the tileservice that {kib} uses
-to display map tiles in tilemap visualizations. By default,
-{kib} reads this URL from an external metadata service, but users can
-override this parameter to use their own Tile Map Service. For example:
+ | The URL to the service that {kib} uses
+as the basemap in all Maps visualizations. By default,
```

Comment:
How about changing to "as the default basemap in <<maps, maps>> and <<vega-with-a-map, vega maps>>"?

Reply:
I was doubting about the wording because we also have the Maps embeddable for SIEM, ML, etc. but I also wanted to avoid too much verbosity. Maybe your proposal is a compromise in between, thanks for the suggestion.